htmlSanityCheck-core/src/test/groovy/org/aim42/htmlsanitycheck/check/BrokenHttpLinksCheckerSpec.groovy

```diff
@@ -29,26 +33,54 @@ class BrokenHttpLinksCheckerSpec extends Specification {
             .withMappingFromResource("mappings.json")
             .withExposedPorts(8080)
 
+    @Shared
+    CustomHostNameResolver customHostNameResolver = new CustomHostNameResolver()
+
     /** executed once before all specs are executed **/
     def setupSpec() {
         wireMockServer.start()
         port = wireMockServer.getMappedPort(8080)
-    }
-
-    /** executed once after all specs are executed **/
-    def cleanupSpec() {
-        wireMockServer.stop()
+        registerCustomDnsResolver()
     }
 
     /* executed before every single spec */
-
     def setup() {
         myConfig = new Configuration()
         brokenHttpLinksChecker = new BrokenHttpLinksChecker( myConfig )
 
         collector = new SingleCheckResults()
     }
 
+
+    /** executed once after all specs are executed **/
+    def cleanupSpec() {
```

Comment:
Just a remark: This should have been part of the previous commit (but I am fine if it stays here).